### DISCUSSION TITLE
Error While Upgrade Sqlite Schema

### DISCUSSION THREAD

**Discussion by BiblKumar** [textbrowser/biblioteq#274] [General]
Hi,
Recently I had downloaded new Biblioteq version and tried to connect to my local_db. After getting connected tried to Upgrade schema but I am getting the error which is attached below and not able to populate members as well. Please can you help me out on this issue.
[Bibloteq Upgrade error.docx](https://github.com/textbrowser/biblioteq/files/10454460/Bibloteq.Upgrade.error.docx)

**Comment by textbrowser** [comment]
The errors indicate that the tables already exist.

**Comment by textbrowser** [comment]
https://github.com/textbrowser/biblioteq/issues/186

**Comment by textbrowser** [comment]
https://github.com/textbrowser/biblioteq/issues/37

**Comment by textbrowser** [comment]
https://github.com/textbrowser/biblioteq/discussions/265

**Comment by textbrowser** [comment]
As I don't know the state of your database and software, I cannot help unless I resuscitate all of the archived database statements and execute them on your database.

**Reply by BiblKumar** [reply]
Thank you so much for all of them who contributed to make the Biblioteq better and better.

Sorry, I didn't follow the release notes. Anyway, I have run the database from older version to newer version, now it is working.

But, I have noticed one issue when I am searching the book using the title in the search option in the main window. I had created the book title as 'Then She was Gone' in the data and when I try to search the book title as 'then she was gone', it is not showing up in the main window. I have to type same case letters as I created to show up. Is it possible to make search option flexible if we type the letters in either uppercase or lowercase. please find the attached screenshot.
![Main Window Error](https://user-images.githubusercontent.com/89518086/213639938-d66723eb-572f-4e52-a948-410215dd8024.png)

I have one more suggestion. Please can you add an option to download the CSV file in the Member's Reservation History.

**Reply by textbrowser** [reply]
The answer is right in front of you... For case insensitivity.

**Comment by textbrowser** [comment]
As for another CSV export. Perhaps you can print it as a PDF. It doesn't interest me, another export.

**Comment by textbrowser** [comment]
Yes, you can print the history as a PDF file.

**Comment by textbrowser** [comment]
Completed export of member history.

https://github.com/textbrowser/biblioteq/commit/f3cef852475773f17e9651cc87e88d0979a75c7e
